components/inputs/docs/input-textarea.md

```diff
@@ -0,0 +1,81 @@
+# TextAreas
+
+The `<d2l-input-textarea>` is a wrapper around the native `<textarea>` element that provides auto-grow and validation behaviours. It's intended for inputting unformatted multi-line text.
+
+![example screenshot of text input](../screenshots/textarea.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-textarea.js';
+</script>
+<d2l-input-textarea
+  label="Description"
+  placeholder="Description of your topic."
+  value="Some description..."></d2l-input-textarea>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `aria-invalid` | String | Indicates that the `textarea` value is invalid |
+| `description` | String | A description to be added to the `textarea` for accessibility |
+| `disabled` | Boolean | Disables the `textarea` |
+| `label` | String, required | Label for the `textarea` |
+| `label-hidden` | Boolean | Hides the label visually (moves it to the `textarea`'s `aria-label` attribute) |
+| `max-height` | String, default: 12rem | Maximum height of the input before scrolling. `none` allows `textarea` to grow infinitely. |
+| `min-height` | String, default: none | Minimum height of the input. If `min-height` and `max-height` are equal then auto-grow will be disabled. |
+| `maxlength` | Number | Imposes an upper character limit |
+| `minlength` | Number | Imposes a lower character limit |
+| `placeholder` | String | Placeholder text |
+| `required` | Boolean | Indicates that a value is required |
+| `value` | String, default: `''` | Value of the `textarea` |
+
+**Accessibility:**
+
+To make your usage of `d2l-input-textarea` accessible, use the following properties when applicable:
+
+| Attribute | Description |
+|--|--|
+| `aria-invalid` | [Indicate that the `textarea` value is invalid](https://www.w3.org/WAI/PF/aria/states_and_properties#aria-invalid) |
+| `aria-label` | Use when `label` does not provide enough context. Only applies if no `label-hidden`. |
```

Comment:
This shouldn't be here, ya? Like `label` + `label-hidden` = `aria-label` I thought.

Reply:
This is true, but it falls back to `aria-label` on the component if neither are specified.  This is same as `d2l-input-text`.  We do mention that `label` is required, so I feel like this is overkill... it's shouldn't actually get used this way.